fairscale/optim/oss.py

```diff
@@ -304,19 +304,6 @@ def clip_grad_norm(
         return total_norm
 
     # State dict interfaces
-    def local_state_dict(self) -> dict:
```

Comment:
We are removing a public API so could potentially break user code. Did anyone ever use this API?

Reply:
I asked, not on the Lightning and HuggingFace side

Reply:
If we were worried, you could just add a deprecation warning for the next release and call the underlying function?

Reply:
good point, I'll do that